platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/files/DataFileCollection.java

```diff
@@ -793,38 +873,14 @@ public interface LoadedDataCallback {
     // =================================================================================================================
     // Private API
 
-    /** Finish a merge file and close it. */
-    private void closeCurrentMergeFile(
-            final DataFileWriter<D> newFileWriter,
-            final CASable index,
-            final ThreeLongsList movesMap,
-            final Semaphore mergingPaused)
-            throws IOException, InterruptedException {
-        // close current file
-        final DataFileMetadata metadata = newFileWriter.finishWriting();
-        // add it for reading
-        final DataFileReader<D> dataFileReader = addNewDataFileReader(newFileWriter.getPath(), metadata);
-        // apply index changes
-        movesMap.forEach((key, oldValue, newValue) -> {
-            try {
-                mergingPaused.acquire();
-                index.putIfEqual(key, oldValue, newValue);
-            } finally {
-                mergingPaused.release();
-            }
-        });
-        // we have updated all indexes now so can now include this file in future merges
-        dataFileReader.setFileAvailableForMerging(true);
-    }
-
     /**
      * Used by tests to get data files for checking
      *
      * @param index data file index
      * @return the data file if one exists at that index
      */
     DataFileReader<D> getDataFile(final int index) {
-        final ImmutableIndexedObjectList<DataFileReader<D>> fileList = indexedFileList.get();
+        final ImmutableIndexedObjectList<DataFileReader<D>> fileList = this.dataFiles.get();
```

Comment:
`this` is unnecessary here and inconsistent with other usage in this class.

Reply:
I fixed a few places like this, but a few more left. Thank you for the catch. I'll get rid of "this" here.